style.css

```diff
@@ -97,19 +102,95 @@ h4 {
   cursor: pointer;
 }
 
+.close-menu {
+  display: none;
+  position: fixed;
+  top: 20px;
+  right: 20px;
+  height: 40px;
+  width: 40px;
+}
+
+.show {
+  display: flex;
+}
+
+.hide {
+  display: none;
+}
+
 .menu {
   display: flex;
-  align-items: center;
   padding-right: 8.53%;
+  position: relative;
 }
 
-.menu p {
+.presentation .menu.active {
+  display: flex;
+  position: fixed;
+  top: 0;
+  left: 0;
+  flex-direction: column;
+  height: 100vh;
+  width: 100%;
+  background-color: #6070ff;
+  z-index: 1;
+  opacity: 0.9;
+}
```

Comment:
[Required] Good job implementing the popup menu, I've noticed that the background design of the menu is not consistent with the Figma template please see the screenshot below for more details on this. The background of the menu should be blurred so that the contents of the page don't obstruct the menu items on the menu. You can achieve this using the following style rules
```CSS
.presentation .menu.active {
  display: flex;
  position: fixed;
  top: 0;
  left: 0;
  flex-direction: column;
  height: 100vh;
  width: 100%;
  background-color: #6070ff;
  z-index: 1;
  background-color: rgba(50, 70, 255, 0.8);
  backdrop-filter: blur(5.5px);
}
```
## Figma template
<img src="https://user-images.githubusercontent.com/16392046/201779877-a4081405-68be-4b59-b603-296a8f8d28e9.png" width="300px" />

## Your current implementation

<img src="https://user-images.githubusercontent.com/16392046/201780310-9a764fbe-6412-4fdb-a671-099ecb45995e.png" width="300px" />